nengo/dists.py

```diff
@@ -221,18 +221,32 @@ class UniformHypersphere(Distribution):
     surface : bool, optional (Default: False)
         Whether sample points should be distributed uniformly
         over the surface of the hyperphere (True),
-        or within the hypersphere (False).
+        or within the hypersphere (False). Defaults to False.
+    min_magnitude : Number, optional (Default: 0)
+        Lower bound on the returned vector magnitudes (i.e.
+        magnitude >= min_magnitude). Must be in the range [0, 1]; defaults
+        to 0. Values outside this range will be clipped to this range. Ignored
+        if ``surface == True``
     """
 
     surface = BoolParam('surface')
+    min_magnitude = NumberParam('min_magnitude', low=0, high=1)
 
-    def __init__(self, surface=False):
+    def __init__(self, surface=False, min_magnitude=0):
         super(UniformHypersphere, self).__init__()
+        self.min_magnitude = np.clip(min_magnitude, 0, 1)
```

Comment:
I agree, and I think this is what caused @jgosmann's confusion with the docstring.